src/chrome/content/rules/Freifunk_Gera-Greiz.xml

```diff
@@ -0,0 +1,17 @@
+<ruleset name="Freifunk Gera-Greiz">
```

Comment:
`Freifunk Gera-Greiz` is the community and `Bürgernetz Gera-Greiz` is the association which was founded to provide a legal entity for the funding of the state 'Thüringen'.